src/promptflow/promptflow/_utils/tool_utils.py

```diff
@@ -129,3 +129,9 @@ def get_inputs_for_prompt_template(template_str):
 def get_prompt_param_name_from_func(f):
     """Get the param name of prompt template on provider."""
     return next((k for k, annotation in f.__annotations__.items() if annotation == PromptTemplate), None)
+
+
+def is_custom_tool_package(module_name: str):
```

Comment:
double confirm if we regard vector_db is custom tool package?

Reply:
The condition looks unnecessary.

Reply:
removed